Dockerfile_arm32

```diff
@@ -1,20 +1,20 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-bionic-arm32v7 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0-disco-arm32v7 AS build
```

Comment:
should it not run on spmething lighter than ubuntu?

Reply:
there are only Debian/Ubuntu arm32 images available. We may decrease the image size by roughly 7MB by running it on Debian.